printable-recipe-generator/src/main.ts

```diff
@@ -78,6 +90,6 @@ if (!process.argv[2]) {
 	//Get chefs and render html
 	void (async () => {
 		const chefsList = (await getChefs()) as Record<string, ChefData>;
-		renderJsonToHtml(process.argv[2], chefsList);
+		renderJsonToHtml(process.argv[2], chefsList).catch(console.error);
```

Comment:
if you're in an async block already you might as well just do `await renderJsonToHtml` rather than using `catch` here. The advantage of _not_ catching is that this should show up as a failed invocation in AWS.